S2VX.Desktop/story.json

```diff
@@ -5,15 +5,15 @@
       "StartTime": 0,
       "EndTime": 10000,
       "StartValue": {
-        "R": 1,
-        "G": 0,
-        "B": 0,
+        "R": 0.5,
+        "G": 0.5,
+        "B": 0.5,
         "A": 1
       },
       "EndValue": {
-        "R": 1,
-        "G": 1,
-        "B": 0,
+        "R": 0.420,
+        "G": 0.727,
+        "B": 0.69,
         "A": 1
       }
     },
```

Comment:
So we can work with it easier, let's add a timingchange into the story.json

Reply:
Ok